Gemfile

```
@@ -10,7 +10,7 @@ gem "rails", "~> 7.0.7", ">= 7.0.7.2"
 gem "sprockets-rails"
 
 # Use postgresql as the database for Active Record
-gem "pg", "~> 1.1"
+gem "pg", "~> 1.2"
```

no need to add any version
```suggestion
gem "pg"
```